### PR TITLE
[AppService] az webapp config ssl sync: Add command to sync imported KV certificate on webapp. 

### PR DESCRIPTION
**Pull request by singlessis**
**Description**  
This PR is adding a command to az webapp config ssl that allows customer to sync an imported KV cert. It effectively makes a create call, and updates the certificate in the app service database with whatever is in the KV. 
**Testing Guide**  

az webapp config ssl sync --resource-group testWestEurope --name SofiTestWestEurope --certificate-name testWestEurope-keyvaultSofiTest-test2048 --key-vault testKV

This certificate, if it is already added to that webapp, will be synced with whatever is in KV. If there is a new cert in KV, we will update the version in our system. 

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AppService

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -2408,6 +2408,37 @@ def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certifi
                                                 certificate_envelope=kv_cert_def)
 
 
+def sync_kv_cert(cmd, resource_group_name, name, certificate_name, key_vault):
+    Certificate = cmd.get_models('Certificate')
+    client = web_client_factory(cmd.cli_ctx)
+    webapp = client.web_apps.get(resource_group_name, name)
+    if not webapp:
+        raise CLIError("'{}' app doesn't exist in resource group {}".format(name, resource_group_name))
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py:2416]: CLIError is deprecated. @houk-ms to provide more details.

Comment (houk-ms) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:2416]: Hi @singlessis,  In this case, it's recommended to use [ResourceNotFoundError](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/azclierror.py#L148) for better error categorizing. 

```
from azure.cli.core.azclierror import ResourceNotFoundError

raise ResourceNotFoundError(' ... your error message ...')
```

See the [error handling guidelines](https://github.com/Azure/azure-cli/blob/e2c77ea31f61a32969abf6d2843b88ba038c4548/doc/error_handling_guidelines.md) for more details, we will anounce it soon.

**Comment by qwordy** [comment]
Please fix linter error. Thanks.

**Pull request closed without merging**